File: CS/BindChartToXPO/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.Xpo;

namespace BindChartToXPO {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {

            for (int i = 1; i < 11; i++) {
                SeriesRecord record = new SeriesRecord();
                record.Argument = "Item " + i.ToString();
                record.Value = i;
                xpCollection1.Add(record);
            }

        }
    }

    public class SeriesRecord : XPObject {
        public string Argument;
        public int Value;
    }

}

File: VB/BindChartToXPO/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Xpo

Namespace BindChartToXPO
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load

            For i As Integer = 1 To 10
                Dim record As New SeriesRecord()
                record.Argument = "Item " & i.ToString()
                record.Value = i
                xpCollection1.Add(record)
            Next i

        End Sub
    End Class

    Public Class SeriesRecord
        Inherits XPObject

        Public Argument As String
        Public Value As Integer
    End Class

End Namespace
